app/views/events/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<p id="notice"><%= notice %></p>
+
+<% if current_user %>
+  <%= link_to 'Log Out', logout_path %><br>
+  <h3>Signed in as: <%= link_to current_user.name.capitalize, user_path(current_user) %></h3>
+<% else %>
+  <%= link_to 'Sign Up', signup_path %> or 
+  <%= link_to 'Log In', login_path %>
+<% end %><br>
```

Comment:
- [ ] Create a helper method to put the if/else statement and create two partials to put the contents of the if and else respectively, render the partials within the if and else statements in the helper method respectively and call the method here instead of having the complexity straight in the view.